notebooks/203-meter-reader/203-meter-reader-digital.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<blockquote>Suggested changes:</blockquote><blockquote>note: The concepts of 'DESIGN_SHAPE' and 'DESIGN_LAYOUT' refer to the fact that the layout information of a display screen is determined during design, and these information is independent of the image. Therefore, you can obtain 'DESIGN_LAYOUT' information from your device manual.&nbsp;However, a simple way to run the program is to treat the coordinates of some points in figure as 'DESIGN_LAYOUT'. You can use <code>Paint</code>, a windows application, to obtain the coordinates. You only need to move the mouse to a position on the image, and the tool will display the corresponding coordinates.</blockquote><blockquote><br></blockquote><p>Can you provide a link to the "Paint" here, so that other developers could directly link to this app and obtain the DESIGN_LAYOUT information？</p><p> </p><p>For "DESIGN_LAYOUT", when you mention "the layout information of a display screen is determined during design, and these information is independent of the image", so it means the area of the LCD screen on the digital meter (the coordinates of the four vertices of the rectangle area), right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1030/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Sorry for replying late. 

> For "DESIGN_LAYOUT", when you mention "the layout information of a display screen is determined during design, and these information is independent of the image", so it means the area of the LCD screen on the digital meter (the coordinates of the four vertices of the rectangle area), right?

about the above question, the answer is right.